src/Analysis/SharedUtils/TeamProject/TeamProjectHeader/TeamProjectHeader.jsx

```diff
@@ -11,19 +12,23 @@ const TeamProjectHeader = ({ showButton }) => {
   const showModal = () => {
     setIsModalOpen(true);
   };
+  const history = useHistory();
 
   useEffect(() => {
     const storedTeamProject = localStorage.getItem('teamProject');
     if (storedTeamProject) {
       setBannerText(storedTeamProject);
-    } else {
+    } else if (showButton) {
```

Comment:
minor naming issue: does it make sense to change `showButton` to `isModifiable`? I think it will help improve the readability.

Reply:
Excellent idea, updated variable name to ```isEditable``` in all files.